lib/compilers/rust.js

```diff
@@ -39,6 +39,8 @@ export class RustCompiler extends BaseCompiler {
         this.compiler.supportsIntel = true;
         this.compiler.supportsIrView = true;
         this.compiler.supportsRustMirView = true;
+        // Macro expansion through -Zunpretty=expanded is only available for Nightly
+        this.compiler.supportsRustMacroExpView = info.name === 'nightly' || info.semver === 'nightly';
```

Comment:
Using this, the "Add Rust Macro Expansion output" button is only visible under the Nightly compiler, making it hard for users to discover the feature.

Ideally, we would want a "show macro expansion output" button like the Rust playground has, which will just fire up the nightly compiler regardless of which one is currently selected.

Reply:
Right, but using a different version of the compiler means that (if) a rust macro expansion changes between versions, things could get confusing. This is more honest (though a popup warning iff we had to pick a newer Rust would work too)